.github/workflows/semantic-release.yml

```diff
@@ -0,0 +1,22 @@
+name: Semantic release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Node 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Install dependencies
+        run: npm ci
+      - name: Lint and test
+        run: npm t
+      - name: Release
```

Comment:
I think we need to pass secrets here, not sure it works like this by default.

https://github.com/sapegin/mrm/blob/d2d81c1d8c42b236b7a42909b24e1dc9bba8cd10/packages/mrm-task-semantic-release/index.js#L9-L43

Reply:
Yes, you're right.
I've added both `GITHUB_TOKEN` and `NPM_TOKEN` as environment variables (you will need to configure them as secrets on GitHub if not already done)